plugin/hooks.go

```diff
@@ -209,4 +210,10 @@ type Hooks interface {
 	//
 	// Minimum server version: 5.30
 	ReactionHasBeenRemoved(c *Context, reaction *model.Reaction)
+
+	// NotificationWillBeSent is invoked before a notification is sent.
+	// To modify the notified user map, return the replacement map,
+	//
+	// Minimum server version: 5.30
```

Comment:
That need to be 5.31 now
```suggestion
	// Minimum server version: 5.31
```